src/web/src/components/SignUp/Schema/FormModel.tsx

```diff
@@ -0,0 +1,55 @@
+export default {
+  displayName: {
+    name: 'displayName',
+    label: 'Display Name',
+    requiredErrorMsg: 'Display Name is required',
+  },
+  firstName: {
+    name: 'firstName',
+    label: 'First name',
+    requiredErrorMsg: 'First name is required',
+  },
+  lastName: {
+    name: 'lastName',
+    label: 'Last name',
+    requiredErrorMsg: 'Last name is required',
+  },
+  email: {
+    name: 'email',
+    label: 'Email',
+  },
+  github: {
+    name: 'github',
+    label: 'Github Data`',
+    invalidErrorMsg: 'Invalid GitHub profile',
+  },
+  githubUsername: {
+    name: 'githubUsername',
+    label: 'Github username',
+    requiredErrorMsg: 'Github account is required',
+  },
+  githubOwnership: {
+    name: 'githubOwnership',
+    label: 'I declare I’m the owner and the maintainer of this GitHub account',
+    invalidErrorMsg: 'You must be the owner of this account',
+  },
+  blogUrl: {
+    name: 'blogUrl',
+    label: 'Blog URl',
+    requiredErrorMsg: 'Blog Url is required',
```

Comment:
`Url` -> `URL` for consistency with the message below.

Reply:
Done.